### DISCUSSION TITLE
how to use _intersects for points inside a polygon?

### DISCUSSION THREAD

**Discussion by preetamslot** [directus/directus#14472] [Archive]
I have created a map where points are displayed within a polygon.
If the user zooms or pans I convert the map bounds to a new polygon and make a new API call to return points in this new search area.

The initial polygon shows all points, but if I zoom in or pan, some points are not returned.
for debugging I render the new search area (polygon) to the map.

Can Someone tell me what I do wrong? Could it be a projection issue or am I using it wrong?
Have checked the search area on the custom endpoint and that gets the right data.

initial screen:
![image](https://user-images.githubusercontent.com/5420582/179358081-c4b9cc0b-3319-4079-b234-13702afbbd1a.png)

after zoom in (no points, should have 2):
![image](https://user-images.githubusercontent.com/5420582/179358101-d59e7e38-830c-4215-88c7-6a984bcd90d0.png)

then a zoom-out (you can see that in the previous search area there are 2 points):
![image](https://user-images.githubusercontent.com/5420582/179358122-df9352a6-081c-4cfb-92ad-7dbb7de1a837.png)

**the filter:**

```
  if (searchArea) {  
      filters.geo = {
        _intersects: JSON.parse(searchArea),
      };
    }
```

**geo object of the points:**
```
{"type":"Point","coordinates":[1.54572,39.00703]}
```

**search area based on the map bounds:**
```
{
    "type": "Polygon",
    "coordinates": [
        [
            [
                1.36745,
                38.89451
            ],
            [
                1.53156,
                38.89451
            ],
            [
                1.53156,
                39.07438
            ],
            [
                1.36745,
                39.07438
            ],
            [
                1.36745,
                38.89451
            ]
        ]
    ]
}
```

**Comment by rijkvanzanten** [comment]
Can you try it with `_intersects_bbox` instead of `_intersects`?
